### PR TITLE
Added information about adding resources in Online Pro

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://trello.com/c/lCjZfK9N/836-docs-document-adding-resources-and-quota-for-online-pro

Preview Build:
http://file.rdu.redhat.com/~ahardin/08242017/adding-resources-online/getting_started/choose_a_plan.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@abhgupta @markturansky PTAL

**Comment by abhgupta** [comment]
Just one minor nit about providing the navigation to get to the "Current Plan" page. Otherwise, lgtm.

**Comment by ahardin-rh** [comment]
@abhgupta Thanks! This is updated.

@openshift/team-documentation Please peer review

File: getting_started/choose_a_plan.adoc [hunk 1]
```diff
@@ -22,6 +22,19 @@ You can also explore link:https://learn.openshift.com/[Red Hat OpenShift's
 Interactive Learning Portal] and link:http://kubernetesbyexample.com/[Kubernetes
 By Example].
 
+[[customize-your-plan-with-add-ons]]
+== Customize Your Plan with Add-ons
+
+If you choose the {product-title} Pro plan, you can customize your plan with
+add-ons.
+
+. In the *My Account* page, click *Change*  by the *Current Plan* heading.
```
Comment (bmcelvee) [inline getting_started/choose_a_plan.adoc]: Just one tiny thing, there's a double space after *Change*

**Comment by bmcelvee** [comment]
Just one nit. Otherwise, LGTM. 🎆

**Comment by ahardin-rh** [comment]
@bmcelvee Thanks! Fixed.

**Comment by ahardin-rh** [comment]
[QE verified.](https://trello.com/c/lCjZfK9N/836-docs-document-adding-resources-and-quota-for-online-pro) No revision history needed.

**Pull request merged**